src/content/blog/introducing-townie.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Introducing Townie AI
+description: Introducing Townie AI
+pubDate: September 25, 2024
+author: Steve Krouse
+---
+
+import { Image } from "astro:assets";
+
+Last month, we launched a major overhaul of Townie, our AI code assistant that helps you build, edit, and deploy your Val Town projects. Val Town is already the best place to turn an idea into a deployed app. Townie makes it even better.
+
+## Townie in Action
+
+Want to see what it's like to build with Townie? Here's a video where I use it to make an RSVP page in just a few prompts and a few minutes. We're talking working, deployed, persistent apps in under 5 minutes. No joke!
+
+<video controls loop class="image-rounded">
+  <source src="/video/TownieIntro.mp4" />
+</video>
+
+But the best way to see what Townie is like is to [just go play with it.](https://val.town/townie)
+
+## 🔥 The Community is on Fire!
```

Comment:
For all of these, I'd probably:

1. Make the image smaller. We'll need to use HTML instead of markdown images
2. Link the image themselves to the post
3. Remove the link in the text. I think the image itself can be the link, maybe